dask_saturn/core.py

```diff
@@ -141,6 +141,10 @@ def _start(
             if not response.ok:
                 raise ValueError(response.reason)
             data = response.json()
+            warnings = data.get("warnings")
+            if warnings is not None:
+                for warning in warnings:
+                    print(f"WARNING: {warning}")
```

Comment:
It'd probably be better to do a proper warning here. I know we currently print rather than logging, but that is a pattern that I'd like to get away from.

Reply:
sure thing

Reply:
going ahead and setting up a logger for everything while I'm at it. You ok with changing to a log format like this, or would you prefer something simpler like `[<log_level>] <message>`?
```
2020-06-24 09:19:44,644 [WARNING] dask-saturn: Unable to update the specification of a running dask cluster. No changes applied.
2020-06-24 09:19:44,644 [INFO] dask-saturn: Cluster is ready

```